packages/dapp/src/components/Widget/Widget.tsx

```diff
@@ -157,12 +163,15 @@ export function Widget({ starterVariant }) {
         },
       },
       keyPrefix: `jumper-${starterVariant}`,
+      ...multisigWidget,
       sdkConfig: {
         apiUrl: import.meta.env.VITE_LIFI_API_URL,
         rpcs,
         defaultRouteOptions: {
           maxPriceImpact: 0.4,
+          allowSwitchChain: !isMultisigSigner, // avoid routes requiring chain switch for multisig wallets
         },
+        multisigConfig: { ...multisigSdkConfig },
```

Comment:
I guess destructuring is not needed here

Reply:
I'd like to create a copy of object wherever sending it. just to avoid any further unwanted updates. Although, it's more of a fail safe thing to do, if it doesn't make sense I can remove it.